app/views/shopping/_recipe_food_row.html.erb

```diff
@@ -0,0 +1,8 @@
+<% qty = RecipeFood.joins(:food).where(food_id: food.id).sum('quantity') # There has to be a better way to do this %>
```

Comment:
**Optional Changes**
- [ ] It is not good practice to write methods in the view. This belongs to the model or at worst to view helper methods. Let us consider refactoring it. You want to keep your view as dumb as possible. Remeber `KISS` principle?